tests/specs/npm/module_field_main/__test__.jsonc

```diff
@@ -0,0 +1,5 @@
+{
+  // use `--node-modules-dir=manual` because the dependency uses a file: spec and local node_modules.
+  "args": "run --quiet --node-modules-dir=manual main.ts",
```

Comment:
This test will fail because the npm package is not installed due to using `--node-modules-dir=manual` (which is the default anyway). An explicit `deno install` would be necessary beforehand or to use `--node-modules-dir=auto` in order to auto install